pages/index.tsx

```diff
@@ -63,15 +63,12 @@ export default function Home() {
       </Head>
 
       <main className="hidden lg:block h-fit w-screen overflow-hidden gap-y-10 dark:bg-neutral-950">
-
-
         <div className="fixed right-0 top-40 z-10">
-        <ConnectWithMe/>
-
+          <ConnectWithMe />
         </div>
 
         <div className="fixed z-40 w-full transition ">
-          <LandingNav />
+          <LandingNav pageNotFound={false} />
```

Comment:
we can just make `pageNotFound` in the component by default `false` so there will be no need to mention it